agent/config_endpoint_test.go

```diff
@@ -213,24 +213,6 @@ func TestConfig_Apply_TerminatingGateway(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, 200, resp.Code, "!200 Response Code: %s", resp.Body.String())
 
-	// Attempt to create an entry for a separate gateway that also routes to web
```

Comment:
Is this test removal for terminating gateways intended here?

Reply:
Yes, this is intentionally removed because we want to support the ability for a service to be defined in more than one gateway. This case was just testing an error occurred when doing so.